.github/workflows/functions-log.yaml

```diff
@@ -1,75 +0,0 @@
-name: functions-log
```

Comment:
This was overridden in template re-run, and got added as standard per-function configs. Per Karl, there were also test failures related to some of the edges of the matrix config setup, such as testing directories without configured tests.

@ace-n  for awareness of this change: besides investigating this for efficiency, was there other considerations in applying the matrix?

Reply:
Discussed this internally - if a parent directory has no tests, we should remove its workflow config *and* ensure its child directories have matching config files.